src/main/resources/assets/colytra/lang/fr_fr.json

```diff
@@ -0,0 +1,15 @@
+{
+  "tooltip.colytra.broken": "Inutilisable",
+  "text.autoconfig.colytra.title": "Colytra",
+  "text.autoconfig.colytra.option.permissionMode": "Mode de permission",
+  "text.autoconfig.colytra.option.permissionMode.@Tooltip[0]": "Défini si la liste de permission est une blacklist ou une whitelist",
+  "text.autoconfig.colytra.option.permissionMode.@Tooltip[1]": "BLACKLIST : Les objets spécifiés ne peuvent être combiné avec des élytres",
```

Comment:
BLACKLIST : **liste noire, l**es objets
ne peuvent **pas** être combiné**s**
avec **une élytre**

Reply:
Per the official translations, elytra are referred as "élytres" and always called with plural pronouns, so that's why they're plural here too.

Reply:
Correct, Elytras are plural as they come in pairs. 
There was still a grammar mistake in this particular block.